lab-andrew/html/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp">
+  <head>
+    <title>Cowsay trolls</title>
+    <link rel="stylesheet" href="bundle.css" type="text/css"/>
+    <script src="bundle.js"></script>
+  </head>
+  <body>
+    <main ng-controller="CowsayController">
+      <form>
+        <label for="cowsay-text">cowsay</label>
+        <input
+        name="cowsay-text"
+        type="text"
+        ng-model="cowsayText"/>
+      <select name="cow files" ng-model="cowsayFile">
+        <option ng-repeat="cow in cowFiles" value="{{cow}}">{{cow}}</option>
+      </select>
+      <button type="submit" ng-click="finalize()">Finalize</button>
+      <button type="submit" ng-click="previous()">Previous - {{count}}</button>
```

Comment:
Change type="submit" on the previous button to type="button".
You aren't submitting the form with this button.